package.xml

```diff
@@ -22,6 +22,10 @@
   <build_depend> real_time_tools </build_depend>
   <run_depend>   real_time_tools </run_depend>
 
+  <!-- This tools provides real time thread objects in an encapsulated way. -->
+  <build_depend> master_board_sdk_catkin </build_depend>
+  <run_depend>   master_board_sdk_catkin </run_depend>
```

Comment:
Just a suggestion: When using format 2 (`<package format="2">`), there is a `<depend>` tag which combines build and run dependency (i.e. no need to duplicate them all).